api/enki.py

```diff
@@ -733,10 +724,8 @@ def update_circulation(self, since):
     def process_book(self, bibliographic):
         """Make the local database reflect the state of the remote Enki
         collection for the given book.
-
         :param bibliographic: A Metadata object with attached
         CirculationData
-
         :return: A 2-tuple (LicensePool, Edition). If possible, a
         presentation-ready Work will be created for the LicensePool.
         """
```

Comment:
I initially thought this was a removal of whitespace, but it's actually removing blank lines in docstrings, which are important information. Was this done automatically by Atom? This isn't a change I want to put into the repo.